package.json

```diff
@@ -73,18 +73,18 @@
     "babel-plugin-transform-react-display-name": "6.25.0",
     "babel-plugin-transform-react-jsx": "6.24.1",
     "babel-plugin-transform-runtime": "^6.23.0",
-    "babel-preset-env": "1.5.1",
+    "babel-preset-env": "^1.7.0",
     "deep-freeze": "0.0.1",
-    "enzyme": "3.2.0",
-    "enzyme-adapter-react-15": "1.0.3",
+    "enzyme": "^3.3.0",
```

Comment:
I'm more comfortable if we pin the exact versions, as to make sure builds are reproducible and prevent subtle bugs.